src/main/java/lotto/view/LottoInputView.java

```diff
@@ -0,0 +1,41 @@
+package lotto.view;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import lotto.domain.exception.IntegerMismatchException;
+import lotto.domain.number.WinningNumbers;
+
+public class LottoInputView {
+
+    private static final Scanner scanner;
+
+    static {
+        scanner = new Scanner(System.in);
+    }
```

Comment:
```suggestion
    private static final Scanner scanner = new Scanner(System.in);
```

이렇게 적으면 더 깔끔할듯 한데요 :)